hack/scale_out_poc/config_haproxy/cfg_generator/cfg_generator.go

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
An updated version of this script has already been moved to master branch in #929. This file should be removed.

Reply:
deleted